frontend/packages/shipwright-plugin/src/components/build-tabsection/BuildOverview.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import * as _ from 'lodash';
```

Comment:
Please avoid lodash for new code if possible. And here it should be not requierd.